config/project-scratch-def.json

```diff
@@ -0,0 +1,18 @@
+{
+  "orgName": "PLP Dev Perview Co",
+  "edition": "Developer",
+  "features": ["PartnerLicensingPlatform"],
+  "settings": {
```

Comment:
yeah these setings below are default, but since we're leaving the file here better to leave it in default state. it's in the same state in our internal test repo also